src/node/rpc/test/membervoting_test.cpp

```diff
@@ -315,49 +338,77 @@ TEST_CASE("Add new members until there are 7, then reject")
       Store::Tx tx;
       ccf::SignedReq sr(proposej);
       Response<Proposal::Out> r =
-        frontend.process_json(rpc_ctx, tx, 0, proposej, sr).value();
+        frontend.process_json(rpc_ctx, tx, proposer_id, proposej, sr).value();
+
       // the proposal should be accepted, but not succeed immediately
       CHECK(r.result.id == proposal_id);
       CHECK(r.result.completed == false);
     }
 
-    Script vote_ballot(R"xxx(
+    // read initial proposal, as second member
+    const Response<OpenProposal> initial_read =
+      get_proposal(rpc_ctx, frontend, proposal_id, voter_a);
+    CHECK(initial_read.result.proposer == proposer_id);
+    CHECK(initial_read.result.script == proposal);
+    CHECK(initial_read.result.parameter == new_member.cert);
+
+    // vote as second member
+    Script vote_ballot(fmt::format(
+      R"xxx(
         local tables, calls = ...
         local n = 0
         tables["members"]:foreach( function(k, v) n = n + 1 end )
-        if n < 8 then
+        if n < {} then
```

Comment:
?

Reply:
libfmt substitutes in `max_members`, so this still becomes `if n < 8` before Lua sees it. But if we change `max_members`, the script still works.

Reply:
Oh I see!